src/main/java/uk/nhs/digital/nhsconnect/nhais/mesh/MeshService.java

```diff
@@ -63,8 +69,9 @@ public void scanMeshInboxForMessages() {
             }
             LOGGER.info("Completed MESH mailbox polling cycle");
         } else {
-            LOGGER.info("Could not obtain database lock to run MESH mailbox polling cycle: insufficient time as elapsed " +
-                "since the previous polling cycle or another adaptor instance has already started the polling cycle.");
+            LOGGER.info("Could not obtain database lock to run MESH mailbox polling cycle: insufficient time has elapsed " +
+                "since the previous polling cycle or another adaptor instance has already started the polling cycle. " +
+                "Next scan in {} seconds", TimeUnit.SECONDS.convert(scanIntervalInMilliseconds, TimeUnit.MILLISECONDS));
```

Comment:
I think next scan wont be in interval, but in interval minus the time from last reading. Although I am not sure if we can retrieve it such accurately. So maybe to say: 'Next scan not later than in {} seconds"?

Reply:
Attempts to scan (obtain db lock) mailbox repeat in intervals, but actual scanning can be done after "delay time" has passed. By default it scans every 60 seconds to check if 5 minutes has passed from previous mailbox access.

Reply:
All right, so the scan is not reading of the mailbox but just checking if reading can be proceeded? If so, I am ok with such message.